packages/ember-runtime/lib/mixins/array.js

```diff
@@ -727,7 +727,7 @@ const ArrayMixin = Mixin.create(Enumerable, {
 
     @method rejectBy
     @param {String} key the property to test
-    @param {String} [value] optional value to test against.
+    @param {*} [value] optional value to test against.
```

Comment:
Hmm, I had assumed that this was another property to check against, but it doesn't seem  like it.